src/components/App.js

```diff
@@ -3,10 +3,10 @@ import ButtonPanel from './ButtonPanel';
 
 function App() {
   return (
-    <div className="App">
+    <fragment>
       <Display />
       <ButtonPanel />
-    </div>
+    </fragment>
```

Comment:
### [Optional]
Good job making the changes from the previous reviewer.

perhaps we could shorten the code.

```
    <>
      <Display />
      <ButtonPanel />
    </>
```